PhysicsTools/PatAlgos/python/slimming/customizeMiniAOD_HcalFixLegacy2016.py

```diff
@@ -0,0 +1,140 @@
+    task.add(process.ak4PFJetsCHS)
+    # need also the non-CHS ones as they are used to seed taus
+    process.ak4PFJets = RecoJets.Configuration.RecoPFJets_cff.ak4PFJets.clone()
+    task.add(process.ak4PFJets)
+    process.ak8PFJetsCHS = RecoJets.Configuration.RecoPFJets_cff.ak8PFJetsCHS.clone()
+    task.add(process.ak8PFJetsCHS)
+
+    process.fixedGridRhoAll = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoAll.clone()
+    process.fixedGridRhoFastjetAll = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetAll.clone()
+    process.fixedGridRhoFastjetCentral = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetCentral.clone()
+    process.fixedGridRhoFastjetCentralChargedPileUp = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetCentralChargedPileUp.clone()
+    process.fixedGridRhoFastjetCentralNeutral = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetCentralNeutral.clone()
+    task.add( process.fixedGridRhoAll,
+              process.fixedGridRhoFastjetAll,
+              process.fixedGridRhoFastjetCentral,
+              process.fixedGridRhoFastjetCentralChargedPileUp,
+              process.fixedGridRhoFastjetCentralNeutral )
+
+    process.load("RecoJets.JetAssociationProducers.ak4JTA_cff")
+    task.add(process.ak4JTATask)
+
+    process.load('RecoBTag.Configuration.RecoBTag_cff')
+    task.add(process.btaggingTask)
+
+    process.load("RecoMET.METProducers.PFMET_cfi")
+    task.add(process.pfMet)
+
+
+def cleanPfCandidates(process, verbose=False):
+    task = getPatAlgosToolsTask(process)
+
+    #add producer at the beginning of the schedule
+    process.load("CommonTools.ParticleFlow.pfCandidatesBadHadRecalibrated_cfi")
+    task.add(process.pfCandidatesBadHadRecalibrated)
+
+    replacePFCandidates = MassSearchReplaceAnyInputTagVisitor("particleFlow", "pfCandidatesBadHadRecalibrated", verbose=verbose)
+    replacePFTmpPtrs = MassSearchReplaceAnyInputTagVisitor("particleFlowTmpPtrs", "particleFlowPtrs", verbose=verbose)
+    for everywhere in [ process.producers, process.filters, process.analyzers, process.psets, process.vpsets ]:
+        for name,obj in everywhere.iteritems():
+            if obj != process.pfCandidatesBadHadRecalibrated:
+                replacePFCandidates.doIt(obj, name)
+                replacePFTmpPtrs.doIt(obj, name)
+
+
+    process.load("CommonTools.ParticleFlow.pfEGammaToCandidateRemapper_cfi")
+    task.add(process.pfEGammaToCandidateRemapper)
+    process.pfEGammaToCandidateRemapper.pf2pf = cms.InputTag("pfCandidatesBadHadRecalibrated")
+    process.reducedEgamma.gsfElectronsPFValMap = cms.InputTag("pfEGammaToCandidateRemapper","electrons")
+    process.reducedEgamma.photonsPFValMap      = cms.InputTag("pfEGammaToCandidateRemapper","photons")
+
+
+    #add bugged conditions to GT for comparison
+    process.GlobalTag.toGet.append(cms.PSet(
+        record = cms.string("HcalRespCorrsRcd"),
+        label = cms.untracked.string("bugged"),
```

Comment:
this part is still here. I expect that this will come from the GT.

Reply:
#23154 which adds this payload to 102X_dataRun2_v1 is available since CMSSW_10_2_X_2018-05-08-2300

Reply:
I will fix that.